samples/WOCCatalog/WOCCatalog/AutoLayoutViewController.m

```diff
@@ -107,6 +109,44 @@ - (void)viewDidLoad {
 
     [self.view addConstraints: [NSLayoutConstraint constraintsWithVisualFormat:@"|-[label1]-[label2(label1)]-[label3(label1)]-[label4(label1)]-|" options:NO metrics:nil views:NSDictionaryOfVariableBindings(label1, label2, label3, label4)]];
     [self.view addConstraints: [NSLayoutConstraint constraintsWithVisualFormat:@"V:[topLabel]-[label1]-[label2(label1)]-[label3(label1)]-[label4(label1)]-[button]" options:NO metrics:nil views:NSDictionaryOfVariableBindings(topLabel, button, label1, label2, label3, label4)]];
+
+    UIButton* button2 = [UIButton buttonWithType : UIButtonTypeRoundedRect];
+    [button2 setTitle:@"Add a new label" forState:UIControlStateNormal];
```

Comment:
The change in this file does not appear to match anything related to the pull reuqest.